scripts/upgrade_scripts/upgrade-fluffy.sh

```diff
@@ -4,17 +4,26 @@
 #   upgrade script from flufflys timer   #
 ##########################################
 
-# change this setting to your install directory
-SERVERDIR="/srv/csgo-server/csgo"
+# change this to your csgo directory, if not running from "upgrade_scripts" folder
+SERVERDIR="../.."
```

Comment:
maybe we could do
`# change below to your csgo server root directory
SERVERDIR="/home/csgoserver/csgo/"`
This would've been easier to undertand to people ✌️